src/sail_sv_backend/jib_sv.ml

```diff
@@ -592,17 +604,22 @@ module Make (Config : CONFIG) = struct
     in
     Str.string_match regexp s 0
 
-  let pp_id_string id =
+  let is_sv_global s =
+    match !sv_globals with
+    | None -> failwith "USED is_sv_global BUT sv_globals IS UNSET"
+    | Some smap -> StringSet.mem s smap
+
+  let pp_id_string ?(prepend_globals = true) id =
```

Comment:
I think this prepend_globals flag is pointless because `Config.no_toplevel_globals` handles this.